### ISSUE TITLE
question about then loss function

### ISSUE THREAD

**Issue by blue-xuhu** [Mephisto405/Learning-Loss-for-Active-Learning#8]
hello
i,read your code.i have some question.
Is the loss function unique?
is the way you do about loss function same with Original author？
i am not sure about this  and  the paper don't say it clearly yet

**Comment by Mephisto405** [comment]
Yeah, the loss function I implemented in this repository is exactly the same as the one in the reference paper. Take a look at the formula (2) on page 96. [LL4AL paper link](https://openaccess.thecvf.com/content_CVPR_2019/papers/Yoo_Learning_Loss_for_Active_Learning_CVPR_2019_paper.pdf)
If you mean the 'target loss' for the CIFAR10 classification task, I think the authors use cross-entropy loss according to the 3rd-6th line of page 98. Since the cross-entropy loss is a typical choice for the CIFAR10 classification task.

**Comment by blue-xuhu** [comment]
Thanks for your reply about loss loss function。
I ran into another problem during the experiment。When using the initialized data set for the first training on the paper，The method of this paper and other methods have achieved almost the same accuracy。I refer to your code, tried many times but couldn't do this. The method in this paper uses the target model and lossnet, while other methods only use the target model. So how did they get the same accuracy in the first training?

**Comment by Bardielz** [comment]
> Thanks for your reply about loss loss function。
> I ran into another problem during the experiment。When using the initialized data set for the first training on the paper，The method of this paper and other methods have achieved almost the same accuracy。I refer to your code, tried many times but couldn't do this. The method in this paper uses the target model and lossnet, while other methods only use the target model. So how did they get the same accuracy in the first training?

helllo,did you finish the problem?
